spec/requests/api/v1/targets/destroy_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe 'DELETE api/v1/targets/:id' do
+  let(:user) { create(:user) }
+  let(:headers) { auth_headers }
```

Comment:
I think this is not necessary unless you're overriding the headers in one of the test cases